src/open_producten/producttypen/tests/api/test_producttype.py

```diff
@@ -813,6 +1449,27 @@ def test_read_product_type_prijs(self):
         ]
         self.assertEqual(response.data["prijzen"], expected_data)
 
+    def test_read_externe_verwijzingen_without_config(self):
+        config = ExterneVerwijzingConfig.get_solo()
+        config.zaaktypen_url = ""
+        config.verzoektypen_url = ""
+        config.processen_url = ""
+        config.save()
+
+        product_type = ProductTypeFactory.create()
+        zaaktype = ZaakTypeFactory(product_type=product_type)
+        verzoektype = VerzoekTypeFactory(product_type=product_type)
+        proces = ProcesFactory(product_type=product_type)
+
+        response = self.client.get(self.detail_path(product_type))
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data["zaaktypen"], [{"url": f"/{zaaktype.uuid}"}])
```

Comment:
I do think it might be good to at least at a `logger.warning()` somewhere when this happens to make sure the warnings show up in Sentry, because configuration is missing and the URLs are incorrect when that is the case

Reply:
added within the viewset `get_serializer_context` method